gen/definitions/snmp_vrf.yaml

```diff
@@ -0,0 +1,11 @@
+---
+name: SNMP VRF
```

Comment:
As this refers to a host under a VRF, this should be renamed to `SNMP VRF Host`.